src/pages/experimental/create-flows/index.mdx

```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
 
 [Vikki Paterson](https://github.com/vikkipaterson)
 
+#### Options:
+
 <AnchorLinks>
 
   <AnchorLink>Standard creation</AnchorLink>
   <AnchorLink>Multistep creation</AnchorLink>
-  <AnchorLink>Confirmation</AnchorLink>
+  <AnchorLink>Anatomy</AnchorLink>
 
 </AnchorLinks>
 
 ![Example](images/1.png)
 
-<Caption>Example of a creation pattern</Caption>
+<Caption>Example of a creation pattern.</Caption>
 
 ## Standard creation
 
-Use the standard creation pattern when no stepped guidance is needed and the input fields are directly related to each other. This pattern applies to side panel, multistep, and full-page creation flows. You should also provide advanced options for the user.
+Used in simple creation tasks when no stepped guidance is needed and the input fields are directly related to each other.
 
 ### Modal
 
-If there are only one or two decisions to be made during a creation, use a small, standard modal overlaying the content where the creation was initiated. Avoid scrolling inside a small modal.
+If there are only one or two options to be made during a creation, use a small, standard modal overlaying the content where the creation was initiated. Avoid scrolling inside a small modal.
 
 <Row>
 <Column colLg={8}>
 
-![Example of a standard creation modal](images/2.png)
+![Example of a standard creation modal.](images/2.png)
 
-<Caption>Example of a standard creation modal</Caption>
+<Caption>Example of a standard creation modal.</Caption>
 
 </Column>
 </Row>
 
 ### Side panel
 
-If you have limited screen real estate, consider using a side panel. Side panels keep the user aware of the context in which they are working. You may choose to use either slide in or slide over panels.
+Consider using a side panel to take advantage of the added real-estate and keep the user in context of where they were working. You may choose to use either the slide in or slide over panels.
 
 <Row>
 <Column colLg={8}>
 
-![Example of a side panel creation flow](images/3.png)
+![Example of a side panel creation flow.](images/3.png)
 
-<Caption>Example of a side panel creation flow</Caption>
+<Caption>Example of a side panel creation flow.</Caption>
 
 </Column>
 </Row>
 
-### Full page
+### Full screen
 
-If your creation flow consists of multiple fields and/or supporting imagery, consider the full-page option. Full-page creation flows allow you to take advantage of maximum screen real estate.
+If your creation flow consists of multiple fields and/or supporting imagery, consider the full screen option to take full advantage of the screen’s available real estate.
 
 <Row>
 <Column colLg={8}>
 
-![Example of a full-page creation flow](images/4.png)
+![Example of a full-page creation flow.](images/4.png)
+
+<Caption> Example of a full page creation flow. </Caption>
 
 </Column>
 </Row>
 
 <Row>
 <Column colLg={8}>
 
-![Example of a full-page creation flow](images/5.png)
+![Another example of a full-page creation flow.](images/5.png)
```

Comment:
```suggestion
![Example of a full-page creation flow.](images/5.png)
```

If someone is using a screen reader, using "Another" in the alt text may be contextually confusing.